documentation/collectionruleexamples.md

```diff
@@ -255,6 +368,27 @@ This rule, named "LongRequestDuration", will trigger when 5 requests each take g
   }
 }
 ```
+  
+</details>
+
+<details>
+  <summary>Environment Variables</summary>
+  
+  ```bash
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Trigger__Type="AspNetResponseStatus"
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Trigger__Settings__ResponseCount="3"
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Trigger__Settings__StatusCodes__0="400"
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Actions__0__Name="MyDump"
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Actions__0__Type="CollectDump"
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Actions__0__Settings__Egress="artifacts"
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Actions__0__Settings__Type="Mini"
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Actions__0__WaitForCompletion="true"
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Actions__1__Type="Execute"
+  export DotnetMonitor_CollectionRules__CollectDumpAndExecute__Actions__1__Settings__Path="C:\\Program Files\\Microsoft Visual Studio\\2022\\Preview\\Common7\\IDE\\devenv.exe"
```

Comment:
Given that this is bash, using a Windows path example is a bit out of place.

Reply:
Would you want to switch it to something else? I wanted to keep it consistent with the JSON example, but we could also change that one too if you had a better idea for them

Reply:
I'm torn about this; it's not that big of a deal the way it is, but at the same time Linux and MacOS users won't be able to use the example as-is. I can't really think of a trivial alternative that would work across platforms except for maybe using [dotnet-dump analyze](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-dump#dotnet-dump-analyze), but that requires user interaction.